resources/profiles/AnkerMake.ini

```diff
@@ -31,41 +31,45 @@ default_materials = Generic PLA+ @ANKER; Generic PLA @ANKER; Generic PET @ANKER;
 avoid_crossing_perimeters = 0
 bridge_acceleration = 2500
 bridge_angle = 0
-bridge_flow_ratio = 0.95
-bridge_speed = 150
+bridge_flow_ratio = 1
+bridge_speed = 50
 brim_separation = 0.1
 brim_type = outer_only
 brim_width = 0
 clip_multipart_objects = 1
 complete_objects = 0
 default_acceleration = 2500
 dont_support_bridges = 1
-elefant_foot_compensation = 0.1
+elefant_foot_compensation = 0.2
 ensure_vertical_shell_thickness = 1
 external_perimeter_speed = 150
 external_perimeters_first = 0
 extra_perimeters = 0
 extruder_clearance_height = 30
 extruder_clearance_radius = 45
+extrusion_width = 0.4
+external_perimeter_extrusion_width = 0.44
+
 fill_angle = 45
-fill_density = 15%
-fill_pattern = cubic
+fill_density = 10%
+fill_pattern = grid
 first_layer_acceleration = 2500
 first_layer_acceleration_over_raft = 0
-first_layer_extrusion_width = 200%
+first_layer_extrusion_width = 100%
```

Comment:
First layer extrusion width, when expressed as a percentage, is calculated from the layer height. At the proposed 100%, a typical 0.2mm layer height print would have a first layer width of 0.2mm. I believe this value should be closer to the default of 200%.

Reply:
@ankermake this is a very good point. I strongly agree with this suggestion